src/styles/blocks/cards.scss

```
@@ -0,0 +1,107 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background-color: $white-color;
+
+  transition: 0.3s;
```

don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

Create a variable for transition duration
For example
```suggestion
    transition: color $transitionDuration, transform $transitionDuration;
```